priv/posts/eef/20220225000000_election-2022.md

```diff
@@ -0,0 +1,112 @@
+
+None of these roles are compensated; this is a volunteer effort because you care to steer the foundation in a direction you believe is important. You may also put the role of board member on your LinkedIn page or resume if that suits you.
+
+## Who are the Outgoing Board Members?
+
+While the founding members assigned themselves to be part of the board as part of the initial bootstrapping effort, the board has been randomly divided into three cohorts, `A` (containing 3 members), `B` (4 members), and `C` (4 members). To ensure continuity, only one cohort is re-elected each year. This means that you will be elected for 3 years at a time.
+
+Here are the cohorts as defined until the next election:
+
+<ul>
+    <li><code class="inline">A</code>, to be re-elected in 2022, 2025, and every three years after:
+        <ul>
+            <li>Richard Carlsson</li>
+            <li>Kenneth Lundin</li>
+            <li>Maxim Fedorov</li>
+        </ul>
+    </li>
+
+    <li><code class="inline">B</code>, to be re-elected in 2024, 2027, and every three years after:
+        <ul>
+            <li>Miriam Pena</li>
+            <li>Peer Stritzinger</li>
+            <li>Sophie DeBenedetto</li>
+            <li>Bryan Paxton</li>
+        </ul>
+    </li>
+
+    <li><code class="inline">C</code>, to be re-elected in 2023, 2026, and every three years after:
+        <ul>
+            <li>Sebastian Strollo (current Secretary)</li>
+            <li>Fred Hebert</li>
+            <li>Alistair Woodman (current President)</li>
+            <li>Francesco Cesarini</li>
+        </ul>
+    </li>
+</ul>
+
+For this years election, the members of cohort `A` are to be re-elected, so the seats of Richard, Kenneth, and Maxim are up for grabs.
+
+## How to Run?
+
+If you want to run, you must be a voting member. Voting members are members who have done one or more of the following:
+
+- A paying member [paying their yearly membership or having paid for a lifetime membership](https://members.erlef.org/join-us)
+- A contributing member, who has spent about 5 hours monthly working in a working group, and who has contacted us at [voting@erlef.org](mailto:voting@erlef.org) to claim their voting rights
+- A managing member, who leads a working group and who has contacted us at [voting@erlef.org](mailto:voting@erlef.org) to claim their voting rights
+- A board member, who has a de-facto membership.
+
+If you want to run, you should contact us at [voting@erlef.org](mailto:voting@erlef.org) with the following information:
+
+- Your name
+- A short biography describing who you are
+- A short text explaining why you think people should vote for you
+- A photo (optional)
+
+If eligible, you will then be added to the [list of candidates](#who-are-the-current-candidates) on this page and on the ballots.
```

Comment:
This goes nowhere. Do you think we need a little place holder on the same page going "we will add candidates here as they submit their candidacy, check back later" or something?

Reply:
Ah yeah I should have left in the headline with a empty section

Reply:
Added the placeholder